test/modules/core/shaderlib/project/project-32-64-glsl.spec.js

```diff
@@ -0,0 +1,326 @@
+    title: 'LNGLAT mode',
+    params: {
+      viewport: TEST_VIEWPORT,
+      coordinateSystem: COORDINATE_SYSTEM.LNGLAT
+    },
+    tests: [
+      {
+        name: 'project_position_to_clipspace_world_position',
+        // NOTE: disbaling transpilation due to https://github.com/stackgl/glsl-transpiler/issues/38
+        disableTranspile: true,
+        func: ({project_position_to_clipspace}) => {
+          const worldPosition = [];
+          project_position_to_clipspace([-122.45, 37.78, 0], [0, 0], [0, 0, 0], worldPosition);
+          return worldPosition;
+        },
+        output: TEST_VIEWPORT.projectFlat([-122.45, 37.78]).concat([0, 1]),
+        precision: PIXEL_TOLERANCE,
+        gpu64BitPrecision: 1e-7,
+        vs: TRANSFORM_VS.project_position_to_clipspace_world_position(
+          [-122.45, 37.78, 0],
+          [fp64LowPart(-122.45), fp64LowPart(37.78)]
+        )
+      },
+      {
+        name: 'project_position_to_clipspace',
+        // NOTE: disbaling transpilation due to: https://github.com/stackgl/glsl-transpiler/issues/39
+        disableTranspileFor64: true,
+        skipGPUs: ['Intel'],
+        func: ({project_position_to_clipspace_vec3_vec2_vec3}) =>
+          project_position_to_clipspace_vec3_vec2_vec3([-122.45, 37.78, 0], [0, 0], [0, 0, 0]),
+        mapResult: coords => clipspaceToScreen(TEST_VIEWPORT, coords),
+        output: TEST_VIEWPORT.project([-122.45, 37.78, 0]),
+        precision: PIXEL_TOLERANCE,
+        gpu64BitPrecision: 1e-7,
+        vs: TRANSFORM_VS.project_position_to_clipspace(
+          [-122.45, 37.78, 0],
+          [fp64LowPart(-122.45), fp64LowPart(37.78)]
+        )
+      },
+      {
+        name: 'project_position_to_clipspace (non-zero Z)',
+        // NOTE: disbaling transpilation due to: https://github.com/stackgl/glsl-transpiler/issues/39
+        disableTranspileFor64: true,
+        skipGPUs: ['Intel'],
+        func: ({project_position_to_clipspace_vec3_vec2_vec3}) =>
+          project_position_to_clipspace_vec3_vec2_vec3([-122.45, 37.78, 100], [0, 0], [0, 0, 0]),
+        mapResult: coords => clipspaceToScreen(TEST_VIEWPORT, coords),
+        output: TEST_VIEWPORT.project([-122.45, 37.78, 100]),
+        precision: PIXEL_TOLERANCE,
+        gpu64BitPrecision: 1e-6, // test fails with 1e-7
+        vs: TRANSFORM_VS.project_position_to_clipspace(
+          [-122.45, 37.78, 100],
+          [fp64LowPart(-122.45), fp64LowPart(37.78)]
+        )
+      }
+    ]
+  },
+  {
+    title: 'LNGLAT_EXPERIMENTAL mode - auto offset',
+    params: {
+      viewport: TEST_VIEWPORT_HIGH_ZOOM,
+      coordinateSystem: COORDINATE_SYSTEM.LNGLAT_EXPERIMENTAL
+    },
+    tests: [
+      {
+        name: 'project_position_to_clipspace_world_position',
+        // NOTE: disbaling transpilation due to https://github.com/stackgl/glsl-transpiler/issues/38
+        disableTranspile: true,
+
+        disableProject64: true, // NOTE: works with project32 but not with project64, is it expected?
```

Comment:
@Pessimistress 
`disableProject64: true` , disables test for `project64` when using `Transform`. I have do this for all `COORDINATE_SYSTEM` enum except `COORDINATE_SYSTEM.LNGLAT`, is this expected behavior? All these tests run fine when using `project32`.